site/book/02-concepts/00.md

```diff
@@ -1,51 +1,44 @@
-  specification which is used for schema-aware operations in kpt. For example, when rebasing a
-  locally modified package to a new version, the merge operation is more than simply performing a
-  text-based file merge, it takes the schema into account.
-
-  In contrast, many configuration tools today interleave data and code, for example by embedding a
-  templating language in YAML. As configuration becomes complex, it becomes hard to read and
-  understand. Furthermore, this prevents external tooling from easily consuming such configuration
-  leading to a closed ecosystem.
-
-- **Extensible**: kpt provides a small core machinery and powerful extension mechanisms. There are
-  two main extension mechanisms in kpt: a) kpt can gain semantic understanding of arbitrary KRM
-  resources by consuming OpenAPI documents. kpt core itself does not hardcode any type-specific
-  information. b) kpt can perform arbitrary operations on resources using kpt functions.
-
-- **Packaging**: A package is a bundle of KRM resources representing a useful unit of functionality.
-
-- **Customizing**: Off-the-shelf packages are rarely deployed without any customization. We know
-  from experience working with Kubernetes configuration and with internal systems at Google that
-  there is no one-size-fit-all customization technique. Sometimes the most appropriate customization
-  is to edit a YAML file directly. Sometimes, you want to automate a one-time customization,
-  sometimes you want to repeatedly perform a customization. Sometimes you want to use
-  parameterization, sometimes perform bulk search & replace, sometimes you want to use a
-  general-purpose programming language. kpt enables all these techniques. In contrast, tools that
-  provide just one customization technique (e.g. parameterization) cannot address all these
-  different use cases well and inevitably lead to anti-patterns (e.g. over-parameterization).
-
-- **Validating**: In the era of fast-paced DevOps, we need to reduce the risk of misconfiguration
-  leading to an outage, security vulnerabilities, or non-compliance. Validation goes hand-in-hand
-  with customization and kpt functions can be used to automate both mutation and validation of
-  resources. Example of validator functions include schema-validation, linting, policy enforcement,
-  and security auditing.
-
-- **Applying**: When it comes to deploying a package to a Kubernetes cluster, kpt complements
-  `kubectl`. By keeping an inventory of deployed resources, kpt enables resource pruning, aggregated
-  status and observability, and an improved dry-run experience.
-
-The two fundamental concepts in kpt are packages and functions. Let's define them.
+> A package-centric toolchain that enables a WYSIWYG configuration authoring, automation, and delivery experience and simplifies managing Kubernetes platforms and KRM-driven infrastructure at scale by manipulating declarative Configuration as Data, separated from the code that transforms it.
```

Comment:
Would be good to turn `WYSIWYG` into link pointing to https://en.wikipedia.org/wiki/WYSIWYG

The description reads very long :) but captures the essence and current state. We can iterate on it in follow ups.

One thing: I had to break the sentence into two to parse it:

A package-centric toolchain that enables a WYSIWYG configuration authoring, automation, and delivery experience.

"that"  (or "it")

simplifies managing Kubernetes platforms and KRM-driven infrastructure at scale by manipulating declarative Configuration as Data, separated from the code that transforms it.

Reply:
Good point. For now, changed "and simplifies" to "which simplifies"

Reply:
sounds good!